### DISCUSSION TITLE
How to add --autolaunch and --theme "dark"? Arguments

### DISCUSSION THREAD

**Discussion by fabiobtex** [vladmandic/sdnext#389] [Q&A]
I know Arguments are mostly in settings but I'm missing those 2 features from auto1111. 
The original theme put the  "?__theme=dark" in the end of the url, showing me how to include that in other themes would be enough

--autolaunch  opens the web browser automatically (or tab if it is already open), no need to copy the address or ctrl+click 

Thank you.

**Comment by vladmandic** [comment]
Autolaunch still exists.
Theme shoul be selected from UI settings.

**Reply by fabiobtex** [reply]
> Autolaunch still exists. Theme shoul be selected from UI settings.
Thanks.
I figured out what to do to make them both active now

**Reply by econundrum** [reply]
Can confirm I'm using autolaunch.

**Reply by Ladypoly** [reply]
where do I add --autolaunch?

**Reply by fabiobtex** [reply]
I made the old automatic1111 way: I created a webui-user.bat:

@echo off

set COMMANDLINE_ARGS= --autolaunch --theme "dark"

call webui.bat

Just those lines are enough
